USAGE.md

```diff
@@ -707,6 +707,8 @@ The `file_info` provider has some special provider options that can be passed th
   Default:`'●'`
 - `file_readonly_icon` (string): The icon that is shown when a file is read-only.<br>
   Default:`'🔒'`
+- `path_sep` (string): The separator character in the file path. <br>
+  Default: `/` or `\\` depending on OS 
```

Comment:
```suggestion
  Default: `/` or `\` depending on OS 
```
People might confuse the `\\` for a literal `\\`. It's better to just include only the character without any escaping done to avoid amibuity. Lua will show an unescaped string error if they try to use a singular `\` without a raw string anyway.

Reply:
Good point